agent/05_agent_configure.sh

```diff
@@ -245,7 +249,112 @@ function generate_cluster_manifests() {
   ansible-playbook -vvv \
           -e install_path=${SCRIPTDIR}/${INSTALL_CONFIG_PATH} \
           "${SCRIPTDIR}/agent/create-manifests-playbook.yaml"
+}
+
+function add_haproxy_server_lines() {
+  num_servers=${1}
+  type=${2}
+  port=${3}
 
+  for (( n=0; n<${num_servers}; n++ ))
+  do
+    cat << EOF >> ${WORKING_DIR}/haproxy.cfg
+  server ${type}-$n ${type}-$n.${CLUSTER_DOMAIN}:${port} check inter 1s
```

Comment:
(weird, I'm pretty sure I commented this line but wasn't able to find my comment anymore :confused: , so re-adding it)

Unless required for other reasons, it's possible to use directly the ip of the instance here instead of the dns name: 
```suggestion
  server ${type}-$n ${AGENT_NODES_IPS[i]}:${port} check inter 1s
```

(or `AGENT_NODES_IPSV6` depending on the network stack) That will allow to remove the block https://github.com/openshift-metal3/dev-scripts/pull/1541/files#diff-4111b804ca04676258ffb172752880ebb5b54f9fa5031d5e620d8193e8dd86d8R35-R38, as a further simplification

Reply:
I remember seeing it. I think it got lost after one of the git pushes. Thanks for adding back the comment. I will add this simplification.